apps/client/src/pages/race/templates/race-text/RaceText.template.tsx

```diff
@@ -295,16 +319,32 @@ export function RaceText({
                     indexConverter.isCursorAtChar(charIndex, {
                       cursorsAt: otherPlayerCursors,
                     });
+                  /* Show normal cursor if no invalid chars */
+                  const isVisibleRegularCursor =
```

Comment:
these names should ne `isRegularCursor visible` &`isInvalidCursorVisible`

Reply:
renamed in #114 a2f934c4d69cdfed5c031f3d9174c8e4a76330e6